sophiread/CMakeLists.txt

```diff
@@ -31,17 +31,17 @@ file(COPY ${CMAKE_SOURCE_DIR}/resources/data DESTINATION ${CMAKE_BINARY_DIR})
 add_compile_options(
   -O3
   -std=c++20
-  -march=native
```

Comment:
this helps improve the performance a bit, removing it might lead to decrease in performance.

Reply:
hmm.  For me the '-march=native' raised errors and prevented compilation.  Maybe we should consider one of the solutions described here:

https://stackoverflow.com/questions/65966969/why-does-march-native-not-work-on-apple-m1